components/section/info/README.md

```diff
@@ -11,7 +11,8 @@ $ npm install @schibstedspain/sui-section-info --save
 ```
 
 ## Usage
-Provide an `string` to a `title` prop and wrap a child component as a `content`.
+Provide an `string` to a `title` prop and wrap a child component as a `content`. If no `title` prop is provided or contains an empty string `''` the corresponding Title DOM element won't be rendered.
```

Comment:
That's a false statement regarding the last merged code :( @davecarter

Reply:
Well... not really, because of that: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator, and `""` is a falsy value 😄. Thanks @davecarter let me see the light 🙏

Reply:
Truthy && Falsy values are kinda confusing 😅